frontend/src/lib/components/DateFilter.js

```diff
@@ -27,117 +27,193 @@ export class DateFilter extends Component {
-                        >
-                            Date range
-                        </a>
-                    </span>
-                ) : (
-                    <div className="dropdown-no-close">
-                        <a
-                            style={{
-                                margin: '0 1rem',
-                                color: 'rgba(0, 0, 0, 0.2)',
-                                fontWeight: 700,
-                            }}
-                            href="#"
-                            onClick={e => {
-                                e.preventDefault()
-                                this.setState({ dateRangeOpen: false })
-                            }}
-                        >
-                            &lt;
-                        </a>
-                        <hr style={{ margin: '0.5rem 0' }} />
-                        <div style={{ padding: '0 1rem' }}>
-                            <label className="secondary">From date</label>
-                            <DatePicker
-                                className="form-control"
-                                selected={rangeDateFrom}
-                                maxDate={new Date()}
-                                onChange={date =>
-                                    this.setState({ rangeDateFrom: date })
-                                }
-                            />
-                            <br />
-                            <label className="secondary">To date</label>
-                            <DatePicker
-                                className="form-control"
-                                selected={rangeDateTo}
-                                maxDate={new Date()}
-                                onChange={date =>
-                                    this.setState({ rangeDateTo: date })
-                                }
-                            />
-                            <button
-                                style={{ marginTop: '1rem' }}
-                                className="btn btn-sm btn-outline-success"
-                                onClick={e =>
-                                    this.props.onChange(
-                                        moment(rangeDateFrom),
-                                        moment(rangeDateTo)
-                                    )
-                                }
-                            >
-                                Apply filter
-                            </button>
-                        </div>
-                    </div>
-                )}
-            </Dropdown>
+                            </Select.Option>
+                        )
+                    }),
+
+                    <Select.Option key={'Date range'} value={'Date range'}>
+                        {'Date range'}
+                    </Select.Option>,
+                ]}
+            </Select>
         )
     }
 }
+
+function DatePickerDropdown(props) {
```

Comment:
![image](https://user-images.githubusercontent.com/1727427/79110109-3f952b00-7d69-11ea-87db-75c9e69b6336.png)

This may need a z-index

Reply:
strange. it wasn't doing this for me but I made it explicit now so lmk!

Reply:
![image](https://user-images.githubusercontent.com/1727427/79169302-fcc46900-7de3-11ea-8e1f-6f585e29198d.png)
Still doing this for me weirdly

Reply:
merged master to get the css to load quickly. don't think it would be the issue but just to make sure. can you make a deployment for us to take a look at?

Reply:
Also happens on the deployment. It's because
![image](https://user-images.githubusercontent.com/1727427/79213197-ac2f2900-7e40-11ea-80d1-41d462c79d27.png)
If you set the z-index for `ant-picker-dropdown` to a larger number than that it works. 

I'm on Chrome if that helps

Reply:
i looked into what was happening in mine and it never seems to reach the 100000 ziindex, but I just pushed the picker's z-index to 999999